apps/console/src/features/email-providers/pages/email-providers.tsx

```diff
@@ -427,7 +428,11 @@ const EmailProvidersPage: FunctionComponent<EmailProvidersPageInterface> = (
     };
 
     const handleBackButtonClick = () => {
-        history.push(`${ AppConstants.getDeveloperViewBasePath() }/channels`);
+        history.push(`${ AppConstants.getDeveloperViewBasePath() }/email-and-sms`);
```

Comment:
Shall we add the path to app constants and use a getter..

example : `AppConstants.getPaths().get("EMAIL_AND_SMS")`

Reply:
Fixed